Core/DISET/private/SSLSocketFactory.py

```diff
@@ -4,7 +4,7 @@
 import types
 from DIRAC import S_OK, S_ERROR, gConfig
 from DIRAC.Core.DISET.private.Transports.SSL.FakeSocket import FakeSocket
-from DIRAC.Core.DISET.private.Transports.SSL.SocketInfoFactory import gSocketInfoFactory
+from DIRAC.Core.DISET.private.Transports.SSL.pygsi.SocketInfoFactory import gSocketInfoFactory
```

Comment:
Is it a problem that this imports from `pygsi`? Either way, maybe warrants a comments.

Reply:
This thing is only used by `HTTPDISETConnection` and frankly I have no clue what this is, or if it is used anywhere. But in any case, this will work only for pygsi as of now

Reply:
It is apparently used in the PilotCStoJSONSynchronizer

Reply:
Oh! well spotted I do not know how I missed that one ! 
Then I Will try to understand this bloody undocumented thing (damn you, whoever wrote that).
Also the GatewayService might be a border case I seem to remember. 
There are probably other problems, that only certification (or prod :) ) will show.
Thanks again

Reply:
I I remoevd HTTPDISETConnection, because it can trivially be replaced by requests in the PilotCSToJSONSynchronizer and because SOAPFactory (also removed :-) ) is useless